### PR TITLE
[droid] switch to jni for json and bitmaps

### PR DESCRIPTION
**Pull request by koying**
- Remove dependency on network
- Always available

Also fixes some bugs
Depends on https://github.com/xbmc/xbmc/pull/13330

### CODE REVIEW

File: tools/android/packaging/xbmc/src/XBMCFile.java.in [hunk 1]
```diff
@@ -0,0 +1,99 @@
+package @APP_PACKAGE@;
+
+import android.util.Log;
+
+/**
+ * Created by koyin on 07/01/2018.
```
Comment (da-anda) [inline tools/android/packaging/xbmc/src/XBMCFile.java.in]: typo in your name ;)

File: xbmc/platform/android/activity/JNIXBMCFile.cpp [hunk 2]
```diff
@@ -0,0 +1,134 @@
+/*
+ *      Copyright (C) 2018 Christian Browet
+ *      http://xbmc.org
```
Comment (da-anda) [inline xbmc/platform/android/activity/JNIXBMCFile.cpp]: xbmc.org? ;)

File: tools/android/packaging/xbmc/src/XBMCFile.java.in (hunk 1 above)
Comment (koying) [reply tools/android/packaging/xbmc/src/XBMCFile.java.in]: Yeah, I know. They were created on Android Studio on windows, which truncated my username for reasons only MS know.

I could have put our standard header, but some of those are based on Apache licensed material, so no actual clue which license they have ;)

File: xbmc/platform/android/activity/JNIXBMCFile.cpp (hunk 2 above)
Comment (koying) [reply xbmc/platform/android/activity/JNIXBMCFile.cpp]: Isn't the **XBMC** foundation still "owning" our assets?
All our headers have xbmc.org, afaict

Comment (MartijnKaijser) [reply xbmc/platform/android/activity/JNIXBMCFile.cpp]: No we changed the address to kodi.tv in as many files as we could. Seems some just keep c/p the old headers.

**Pull request merged**